etc/icinga2/features-available/windowseventlog.conf

```diff
@@ -0,0 +1,8 @@
+/**
+ * The WindowsEventLogLogger type writes log information to the Windows Event Log.
+ */
+
+object WindowsEventLogLogger "windowseventlog" {
+    severity = "warning"
```

Comment:
Ok, maybe I'm not only changing the docs. Given that we replace `mainlog`, I think we should have severity `information` here, as that was used for `mainlog` (it's `warning` here as that was just copied from `SyslogLogger`).

Reply:
I actually thought it's nice to have `warning` as the default, because it wouldn't be as much spam. But you're right, this would change the current behavior and agent logs aren't that spammy.